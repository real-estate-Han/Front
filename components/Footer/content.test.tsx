import React from 'react';
import { render } from '@testing-library/react';
import FooterContent from './content';

describe('FooterContent 컴포넌트', () => {
  it('스냅샷이 일치하는지 확인한다.', () => {
    const { asFragment } = render(<FooterContent />);
    expect(asFragment()).toMatchSnapshot();
  });
});
