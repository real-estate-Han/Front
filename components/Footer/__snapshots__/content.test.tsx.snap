// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterContent 컴포넌트 스냅샷이 일치하는지 확인한다. 1`] = `
<DocumentFragment>
  <div
    class="emotion-0"
  >
    <div>
      회사소개 | 의뢰하기 | 이용약관 | 개인정보처리방침
    </div>
    <div>
      <p>
        한세일 부동산
      </p>
      <p>
        상호 : 한세일 부동산|대표자 : 한세일 | 주소 : 경기도 파주시 문산읍
      </p>
      <p>
         부동산등록번호 : 41480-2023-00100
      </p>
      <p>
        전화 : 031-954-6300 | 팩스 : 031-954-1110 | 이메일 : sale@naver.com
      </p>
    </div>
    <div>
      대표전화 call center 031-954-1110
    </div>
  </div>
</DocumentFragment>
`;
